src/scss/packages/_usa-alert.scss

```diff
@@ -4,12 +4,19 @@
 $alerts: 'success', 'warning', 'error', 'info', 'emergency';
 
 .usa-alert {
-  border-radius: radius('lg');
+  &,
+  & .usa-alert__body {
+    border-radius: radius('lg');
```

Comment:
It appears that as of USWDS 3, the alert background is applied to both `.usa-alert` as well as `.usa-alert__body`, so adding the `border-radius` to `.usa-alert` would cause the appearance of squared corners, since `.usa-alert__body` would previously have not had the same border radius.